code/game/machinery/rechargestation.dm

```diff
@@ -255,3 +255,23 @@
 	set src in oview(1)
 
 	go_in(usr)
+
+
+/obj/machinery/recharge_station/MouseDrop_T(var/atom/movable/C, mob/usr)

```

Comment:
`mob/usr)`

Really? No. `usr` is a reserved word, please just find and replace with `user` or literally anything else.